pages/02.Documentation/09.orchestration/docs.en.md

```diff
@@ -479,3 +481,18 @@ outputs:
     description: Floating IP address of loadbalancer in public network
     value: { get_attr: [ loadbalancer_floating_ip, floating_ip_address ] }
 ```
+
+---
+
+## Questions & Answers
+
```

Comment:
the question about ubuntu 15.x default routes is missing

Reply:
see above